### ISSUE TITLE
Explanatory Agent Creative Mode ARS Registration

### ISSUE THREAD

**Issue by tziomics** [NCATSTranslator/Relay#305]
We have finished SmartAPI registration for the creative mode Explanatory Agent instance. Its documentation is at [https://smart-api.info/ui/fe35296eea32e2e829df673c98e36615](https://smart-api.info/ui/fe35296eea32e2e829df673c98e36615). The example /query request will return valid results.

Please register this service with the ARS.

**Comment by MarkDWilliams** [comment]
@tziomics thanks for passing this along.  Taking a look at the registration, I notice that the x-maturity is `staging`.  Set up that way, only the CI instance of the ARS will pull this registration, not the production one.  Is that your intention?

**Comment by tziomics** [comment]
We only have the one instance, currently. Would it be better to set the maturity to production? Or maybe have all of the maturities point to the same instance?

**Comment by MarkDWilliams** [comment]
I'm happy to put the tool in place as is, or with alterations.  I just wanted to make sure you were aware that as it, it would only show up in the ARS CI environment.

**Comment by tziomics** [comment]
Yes, and I think that's the right approach, after thinking about it. This creative mode ARA is just a prototype, so I wouldn't want to mislead anyone with our answers on a production-level query. Please go ahead and register it with just the one x-maturity definition.
